frontend/src/edit.jsx

```diff
@@ -132,44 +176,54 @@ export default function EditMenu(props) {
-                </RadioGroup>
-                <h4>Add this modification to a scenario</h4>
-                <datalist id="scenariolist">
-                  {Object.values(savedScenarios).map(item => <option key={item} value={item} />)}
-                </datalist>
-                <input
-                  type="search"
-                  id="scenarioName"
-                  list="scenariolist"
-                  value={scenarioName}
-                  onChange={(event) => setScenarioName(event.currentTarget.value)}
-                />
-                <button type="submit">Submit</button>
-              </form>
+              {
+                (parcel)
+                  ? (
+                    <form onSubmit={handleSubmitNew}>
+                      <RadioGroup
+                        className="sidebar-subheading"
+                        inline
+                        label="Modify the landuse of this parcel by:"
+                        onChange={handleConversionOption}
+                        selectedValue={conversionOption}
+                      >
+                        <Radio key="wallpaper" value="wallpaper" label="wallpaper" />
+                        <Radio key="paint" value="paint" label="paint" />
+                      </RadioGroup>
+                      <div className="conversion-panel">
+                        {
+                          (conversionOption === 'paint')
+                            ? (
+                              <HTMLSelect onChange={handleSelectLULC}>
+                                {Object.entries(lulcCodes)
+                                  .map(([k, v]) => <option key={k} value={k}>{v}</option>)}
+                              </HTMLSelect>
+                            )
+                            : wallpaperingSelect
+                        }
+                      </div>
+                      <p className="sidebar-subheading">
+                        Add this modification to a scenario
+                      </p>
+                      <datalist id="scenariolist">
+                        {Object.values(savedScenarios)
+                          .map(item => <option key={item} value={item} />)}
+                      </datalist>
+                      <input
+                        type="search"
+                        id="scenarioName"
+                        list="scenariolist"
+                        value={scenarioName}
+                        onChange={(event) => setScenarioName(event.currentTarget.value)}
+                      />
+                      <button type="submit">Submit</button>
+                    </form>
+                  )
+                  : <div />
+              }
             </div>
           )}
         />
-        <Tab
-          id="patterns"
-          title="Patterns"
-          panel={
-            <div>
-              <Switch
-                checked={patternSamplingMode}
```

Comment:
I think moving this into the other tab is fine for now. But eventually we might want more space to display info and preview the different wallpaper patterns.

Reply:
I agree. A dropdown for the patterns will not be sufficient. I think we'll need to display thumbnails. Maybe with a modal dialog that can be as large as we need. 

I like the flow of guiding the user through all the steps without them having to do the navigation on their own though:
1. Select a parcel
2. Select a conversion option
3. Do wallpapering / paint
4. Submit

Reply:
Cool. Maybe eventually replacing the tabs with a vertical accordion - one section corresponding to each step?